node-notes-cli/app.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs');
+const data = require('./data.json');
+const input = process.argv.filter((x, i) => i > 1);
+const command = input[0];
+
+const readNotes = data => {
+  for (const key in data.notes) {
+    console.log(`${key}: ${data.notes[key]}`);
+  }
+};
+
+const saveData = data => {
+  const dataJSON = JSON.stringify(data, null, 2);
+  fs.writeFile('./data.json', dataJSON, err => { if (err) throw err; });
+};
+
+const checkID = id => {
+  if (!(id in data.notes)) {
+    console.log('Not a valid ID');
+    process.exit();
```

Comment:
You should write error messages to `console.error()`. Be sure to exit with a non-zero code (`1` is fine) if your app needs to quit with some kind of error.